contracts/interfaces/IElections.sol

```diff
@@ -1,69 +1,62 @@
 
-	/// @dev Called by a validator when ready to join the committee, typically after syncing is complete or after being voted out
-	function notifyReadyForCommittee() external;
+	/*
+     * External methods
+     */
 
-	/// @dev Stake delegation
-	function delegate(address to) external;
+    /// @dev Called by a validator as part of the automatic vote unready flow
+	function voteUnready(address subject_addr) external;
 
-	/// @dev Called by a validator as part of the automatic vote-out flow
-	function voteOut(address addr) external;
+    /// @dev Called by a validator as part of the vote-out flow
+	function voteOut(address[] calldata subject_addrs) external;
 
-	/// @dev Refreshes the staking information (and the corresponding rank in committee and topology) for the given addresses.
-	function refreshStakes(address[] calldata addrs) external;
+	/// @dev Called by a validator when ready to join the committee, typically after syncing is complete or after being voted unready
+	function readyForSync() external;
 
-	/// @dev casts a banning vote by the sender to the given address
-	function setBanningVotes(address[] calldata addrs) external;
+	/// @dev Called by a validator when ready to join the committee, typically after syncing is complete or after being voted unready
+	function readyForCommittee() external;
 
 	/*
-	 *   Methods restricted to other Orbs contracts
-	 */
+     * Methods restricted to other Orbs contracts
+     */
 
-	/// @dev Called by: staking contract
-	/// Notifies a batch of stake updates
-	function stakeChangeBatch(address[] calldata _stakeOwners, uint256[] calldata _amounts, bool[] calldata _signs,
-		uint256[] calldata _updatedStakes) external /* onlyStakingContract */;
+	/// @dev Called by: delegation contract
+	/// Notifies a delegated stake change event
+	/// total_delegated_stake = 0 if addr delegates to another validator
+	function delegatedStakeChange(address addr, uint256 selfStake, uint256 total_delegated, uint256 delta_total_delegated, bool sign_total_delegated); /* onlyDelegationContract */;
 
-	/// @dev Called by: staking contract
-	/// Notifies an stake change event
-	function stakeChange(address _stakeOwner, uint256 _amount, bool _sign, uint256 _updatedStake) external /* onlyStakingContract */;
+	/// @dev Called by: delegation contract
+	/// Notifies a batch of delegated stake updates - TBD if needed
+	function delegatedStakeChangeBatch(address[] calldata addr, uint256[] calldata selfStake, uint256[] calldata delegated_stake); /* onlyDelegationContract */;    
 
-	/// @dev Called by: staking contract
-	/// Notifies an stake migration event
-	function stakeMigration(address _stakeOwner, uint256 _amount) external /* onlyStakingContract */;
+	/// @dev Called by: validator registration contract
+	/// Notifies a new validator was registered
+	function validatorRegistred(); /* onlyValidatorsRegistrationContract */;    
```

Comment:
missing arguments (validator address, anything else?)

Reply:
Fixed. No need for additional data as the validators registration owns the rest of the data